content/security/security.txt

```diff
@@ -113,6 +113,12 @@ All fields MAY use the following placeholders in the URL templates that are dyna
 
 The `changes` and `download` fields MUST use the `{{ version }}` placeholder.
 
+## `Php` field
+
+The `Php` field contains the end-of-life dates of each major PHP release according to https://www.php.net/supported-versions.php. They can be used to warn users who use an unsupported version.
+
+The key of each entry MUST be set to a string of the major PHP release. The value MUST be set to a ISO 8601 date value.
+
```

Comment:
Could anyone tell me how can I see/test this `Docs` part in website? Where is rendering this `docs` part of `security` page?

Reply:
It isn't rendered anywhere, just internal for us at the moment.

Reply:
Oh, ok 🙂